config/config.go

```diff
@@ -183,3 +187,19 @@ func Load(ctx *cli.Context, loadNetworkConfig bool) (*Config, error) {
 	}
 	return cfg, nil
 }
+
+// NewKeyFromKeystore creates a private key from a keystore file
+func NewKeyFromKeystore(cfg types.KeystoreFileConfig) (*ecdsa.PrivateKey, error) {
```

Comment:
nice to have this here, later we can get rid of the old way of getting this through etherman 😄